app/enterprise/0.33-x/developer-portal/best-practice.md

```diff
@@ -0,0 +1,204 @@
+
+If you are using AWS for you hosting, I would provision a [t2.large](https://aws.amazon.com/ec2/instance-types/t2/) as a starting place. This is a solid base and should be able to handle our expectations without any issue. If you use Google Cloud (GCE), I would suggest starting with a [n1-standard-2](https://cloud.google.com/compute/pricing#standard_machine_types) and if you are using the Azure cloud, I would suggest starting with a [Standard_D2_v3](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general#dv3-series-sup1sup).
+
+If you look at those recommendations, you'll see they all have similar starts:
+  - At least 2 CPUs
+  - Around 8 GiB System memory
+
+I also recommend having this Kong instance be used to only serve the dev portal, and not do any Kong administrative work. To do this, in your kong.conf file, set the following values
+```
+admin_listen=off
+portal=on
+```
+We will also want to ensure all other instances of Kong have the portal boolean set to off. This ensures your portal is only being served by a single Kong instance.
+
+## <a name=css></a>Customizing The Portal Design with CSS
+
+The Developer Portal has two files that contain CSS and are used to control the design. The main CSS file is `unauthenticated/theme-css` which contains the bulk of the CSS styles for the default layout. The second file is `unauthenticated/custom-css` where owners add and edit their own custom css. Both of these files can be accessed by non authenticated pages. You can access the custom-css file by using the Admin API<br />
+`curl http://localhost:8001/files/unauthenticated/custom-css`<br /><br />
+
+This will return the a JSON object that represents the custom-css file. It contains metadata about the file, and the `contents` field contains the actual CSS data. You should save the `contents` to a file, and edit it to your liking. Edit the CSS to customize the look of your portal.
+
+When you have customized the CSS, you will then upload it to the Files API. This will make the Dev Portal use the CSS for when it serves content, and show your custom design. <br />
+`curl -X PATCH http://localhost:8001/files/unauthenticated/custom-css -F contents=@custom.css`<br /><br />
+
+In this example the name of the file on disk is `custom.css`. You are free to name this whatever you like, so long as you send your changes to the `unauthenticated/custom-css` file. This specific place is where the Kong Dev Portal will look for custom CSS.
+
+
+## <a name=js></a>Customizing Front End Behavior with JavaScript
+
+Including custom javascript is similar to customizing CSS.
+
+In order to use custom javascript we need to download the `custom-js` file, make our custom changes, and then upload those changes to the `custom-js` file.
+
+Use this command to get the JSON object that represents the `custom-js` file<br />
+`curl http://localhost:8001/files/custom-js`<br /><br />
```

Comment:
When providing curl commands, please use the same formatting as is present in our other documentation - you can see some examples of that here https://raw.githubusercontent.com/Kong/docs.konghq.com/master/app/0.14.x/getting-started/adding-consumers.md

Reply:
fixed